lib/breakers/uptime_middleware.rb

```diff
@@ -49,7 +49,9 @@ def outage_response(outage:, service:)
     end
 
     def handle_request(service:, request_env:, current_outage: nil)
+      start_time = Time.now
       return @app.call(request_env).on_complete do |response_env|
+        response_env[:duration] = Time.now - start_time
```

Comment:
Let me know if you disagree. It's probably not the end of the world if we don't follow this "convention." I'm happy to approve if you think it's fine as it is.